webpack/webpack.common.js

```diff
@@ -0,0 +1,112 @@
+const path = require('path');
+const webpack = require('webpack');
+BASE_PATH = path.resolve(__dirname, '..')
+
+module.exports = {
+    entry: {
+        tests: {
+            import: './tests/main.js',
+        },
+        main: {
+            import: './src/main.js',
+            chunkLoading: false,
+        },
+    },
+    output: {
+        clean: true,
+        path: path.resolve(BASE_PATH, '_build'),
+    },
+    module: {
+        rules: [
+            {
+                test: /\.xml$/,
+                type: 'asset/source',
+            },
+            {
+                test: /\.html$/,
+                type: 'asset/source',
```

Comment:
I'd prefer to use something like `underscore-template-loader` here, but it breaks a number of tests in ways that I haven't yet figured out. I don't want to block this PR on resolving those.